buildstockbatch/postprocessing.py

```diff
@@ -315,7 +419,7 @@ def combine_results(fs, results_dir, cfg, do_timeseries=True):
 
     # create the postprocessing results directories
     for dr in dirs:
-        fs.makedirs(dr)
+        fs.makedirs(dr, exist_ok=True)
```

Comment:
I will revert this. I added this because at one point I was working on allowing to resume from last failed upgrade. I have removed that capability now (if you need to redo postprocessing, you redo all).